### PR TITLE
Add staffing percentage and staffing in period + target for local db

### PR DESCRIPTION
**Pull request by petternett**
Added staffing percentage to staffing table, which is backwards compatible with existing values because of the default value being set to 100%.
Also changed the primary key to be unique to (employee, date, percentage), so that an employee can be staffed on multiple projects a day.
There is also a new function for adding staffing in a period of time, which will automatically compute the weekdays. Please notice that this does not yet take into account the employee's availability during those dates.

### CODE REVIEW

**Review by ITJohan — CHANGES_REQUESTED** [review]
Nice work! I think the primary key thing needs to be worked out, so not approving yet:)

File: sqitch/deploy/alter_table_staffing_add_staffing_percentage.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+-- Deploy floq:alter_table_staffing_add_staffing_percentage to pg
+
+BEGIN;
+
+ALTER TABLE staffing
+    DROP CONSTRAINT staffing_pkey,
+    ADD COLUMN percentage INT NOT NULL DEFAULT 100,
+    ADD PRIMARY KEY (employee, date, percentage);
```
[the remark this answers is not in the document]
Comment (magnedavidsen) [reply sqitch/deploy/alter_table_staffing_add_staffing_percentage.sql:8]: What ITJohan said. Primary key needs to be employee, project, date – right?

[the remark this answers is not in the document]
Comment (petternett) [reply sqitch/deploy/alter_table_staffing_add_staffing_percentage.sql:8]: right!

File: functions/add_staffing_in_period.sql [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+			FROM date_range
+			WHERE EXTRACT(DOW FROM cur_date) BETWEEN 1 AND 5
+		),
+		new_staffing AS (
+			INSERT INTO staffing (employee, date, project, percentage)
+			SELECT
+				in_employee AS employee,
+				weekdays.cur_date AS date,
+				in_project AS project,
+				in_percentage AS percentage
```
[the remark this answers is not in the document]
Comment (petternett) [reply functions/add_staffing_in_period.sql:25]: we will consider this at a later time, for now let's just try to not input wrong values :p

**Review by ITJohan — APPROVED** [review]
💯

**Pull request merged**